recipes/mrcz/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "mrcz" %}
+{% set version = "0.3.6" %}
+{% set sha256 = "8b97aa55d2827d0d8371f5a713d6c26a68952c97631a2243b698e0fcb4998992" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - blosc
```

Comment:
It doesn't look like you need `blosc` as a `build` requirement since it's not imported via the `setup.py`

Reply:
Working as intended; `blosc` is an optional dependency but as it's available on `conda` and there are tests for when it's installed I would prefer it to be a dependency on `conda-forge`.

Reply:
Just to be clear, if it's listed as a `run` dependency, then when a user installs this package via conda-forge, they will get `blosc`. What I was trying to say is that I don't think you actually need it to `build` the recipe as it's not listed in the `setup_requires` section in `setup.py`, nor is it imported by `setup.py`.

Reply:
> Just to be clear, if it's listed as a run dependency, then when a user installs this package via conda-forge, they will get blosc.

Good, that's what I want.  We don't do this for `pip` because `blosc` has lots of issues with `multiwheels` support with the Snappy codec.